src/rtc_base/physical_socket_server.cc

```diff
@@ -567,7 +567,7 @@ int PhysicalSocket::TranslateOption(Option opt, int* slevel, int* sopt) {
 #elif defined(WEBRTC_MAC) || defined(BSD) || defined(__native_client__)
```

Comment:
it seems this codepath should be in `#else`, actually

Reply:
Agreed